tests/TestCase.php

```diff
@@ -50,30 +50,46 @@ protected function getLineCoverageXdebugDataForBankAccount()
             RawCodeCoverageData::fromXdebugWithoutPathCoverage([
                 TEST_FILES_PATH . 'BankAccount.php' => [
                     8  => 1,
+                    9  => -2,
                     13 => 1,
+                    14 => -1,
+                    15 => -1,
```

Comment:
Reference:

https://github.com/sebastianbergmann/php-code-coverage/blob/caeda52223e37eaddc3ae61e7d4401f4e6821e71/tests/_files/BankAccount.php#L11-L18

### Line 14

Line 14 is **always** reported by each driver. Not having it here can introduce a bug on CC merging in the future

### Line 15

This line is not reported by any driver, but I kept it here because removing it would fail pretty much every test in this library.
I've checked each of them, and removing this reported line to be consistent with drivers output adds no value to the test suite.